orttraining/orttraining/core/graph/gradient_builder.cc

```diff
@@ -1470,5 +1470,13 @@ IMPLEMENT_GRADIENT_BUILDER(GetFlattenGradient) {
   };
 }
 
+IMPLEMENT_GRADIENT_BUILDER(GetTopKGradient) {
+  return std::vector<NodeDef>{
+      NodeDef(OpDef{"TopKGrad", kMSDomain, 1},
+              {GO(0), O(1), I(0)},
+              {GI(0)},
+              SrcNodeAttributes())};
```

Comment:
Shall we make the copy of axis attributed explicit, as don't need the other two attributes.